lib/common/cmn_files.ml

```diff
@@ -0,0 +1,14 @@
+open Core.Std
+open Core_extended.Std
+open Async.Std
+
+exception Cmn_write_error
+
+let write root name contents =
```

Comment:
It should, but those have not been merged yet. When they are I will either update this pr or create a new pr to add them.